### PR TITLE
Dev/sol/sec

### PR DESCRIPTION
**Pull request by SoloJacobs**

### CODE REVIEW

File: src/bin/scheduler/setup/rcc.rs [hunk 1]
```diff
@@ -260,20 +262,20 @@ fn enable_long_path_support(
...
-    command_spec.add_arguments(["holotree", "init"]);
+    command_spec.add_arguments(["holotree", "init", "--revoke"]);
     let mut succesful_plans = vec![];
     let mut failed_plans: HashMap<String, String> = HashMap::new();
 
     for (session, plans) in plans_by_sessions(plans) {
         debug!("Running {} for `{}`", command_spec, &session);
         let session_id = &format!(
-            "holotree_initialization_{}",
+            "holotree_disabling_{}",
```
Comment (jherbel) [inline src/bin/scheduler/setup/rcc.rs:278]: let's also call it `"holotree_disabling_sharing_{}"`

File: tests/test_scheduler.rs [hunk 2]
```diff
@@ -397,19 +396,24 @@ async fn assert_rcc_files_permissions(
...
-    assert!(stdout.contains("holotree-global-shared                ...  \"true\""));
+    let diagnostics = read_configuration_diagnostics(&rcc_config.binary_path)?;
+    assert_eq!(
+        diagnostics
+            .details
+            .get("telemetry-enabled")
+            .unwrap()
+            .as_str(),
+        "false"
+    );
```
Comment (jherbel) [inline tests/test_scheduler.rs:407]: In the last commit, we should explicitly test that the shared holotree is disabled if possbile

File: src/rcc.rs [hunk 3]
```diff
@@ -0,0 +1,242 @@
+use camino::Utf8Path;
```
Comment (jherbel) [inline src/rcc.rs:1]: Since we only use this for testing, let's move it to the test folder

**Review by jherbel — CHANGES_REQUESTED** [review]
Only minor comments, nice work!

File: src/bin/scheduler/setup/rcc.rs (hunk 1 above)
Comment (SoloJacobs) [reply src/bin/scheduler/setup/rcc.rs]: Done

File: tests/test_scheduler.rs (hunk 2 above)
Comment (SoloJacobs) [reply tests/test_scheduler.rs:408]: Done

File: src/rcc.rs (hunk 3 above)
Comment (SoloJacobs) [reply src/rcc.rs]: Done

**Review by jherbel — APPROVED** [review]

**Pull request merged**